### DISCUSSION TITLE
Error during Upload

### DISCUSSION THREAD

**Discussion by Wikojs** [requarks/wiki#5183] [Error / Bug Report] [answered]
My Wiki JS has its latest version, Docker in Ubuntu has it latest version. The group has all 12 global permissions and all 12 page rules permissions.

The path to Site where users upload a picutre is:
/home/Kundenbetreuung/PZE/new-page

So it should be covered with these permissions, right? 
please help

![wiki](https://user-images.githubusercontent.com/102657017/162965481-bb779043-f8b0-4c5e-a772-bb9d067b09f5.png)

![wiki 2 ](https://user-images.githubusercontent.com/102657017/162966931-8b3fdb10-b799-4e57-85ed-85447562f3c1.png)

![wiki 3 ](https://user-images.githubusercontent.com/102657017/162966948-abdf079f-cd45-4994-b519-8f1bdd9328d0.png)

**Comment by Wikojs** [comment, marked as answer]
When creating a new group do not touch edit or delete the first path specification under "page rules"

![wikkk](https://user-images.githubusercontent.com/102657017/162995155-e1953868-daf6-49d9-b831-627f554bd329.png)

**Reply by NGPixel** [reply]
`/home` is a special path that corresponds to the page at `/`, you shouldn't use it as a base for other paths.

Your second rule should be `/Kundenbetreuung`
